### DISCUSSION TITLE
AWS Cloud Services | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1535] [Q&A]
# AWS Cloud Services | Seeed Studio Wiki

Connect SenseCAP T1000 Tracker to AWS

https://wiki.seeedstudio.com/SenseCAP_T1000_Tracker_AWS/

**Comment by mellerbeck** [comment]
You shouldn't need to use AdministratorAccess (this is complete access)

I think AWSIoTDataAccess would be sufficient (And should actually be scoped even more granular....)

--

Click Add permissions → Attach policies.

Search “AdministratorAccess”, check the box left it, and then click “Add Permissions”.
